src/index.js

```diff
@@ -154,24 +143,14 @@ const makeDOMDriver =
         validateDOMDriverInput(view$)
 
         const rootElem$ =
-          most.create(
-            add =>
-              view$
-                .flatMap(parseTree)
-                .reduce(
-                  (prevView, newView) => {
-                    patch(prevView, newView)
-                    add(newView.elm)
-                    return newView
-                  }
-                  , rootElem
-                )
-          )
-        rootElem$.drain()
-
+          view$
+            .map(parseTree)
+            .switch()
+            .scan((oldVnode, vnode) => patch(oldVnode, vnode), rootElem)
+            .skip(1) // emits rootElem first
```

Comment:
If this comment is *really* necessary, then place single line comments on a new line above the subject of the comment, put an empty line before the comment.

```js
// skip(1) because `rootElem` is emitted first
const rootElem$ =
  view$
    .map(parseTree)
    .switch()
    .scan((oldVnode, vnode) => patch(oldVnode, vnode), rootElem)
    .skip(1)
```

Or

```js
const rootElem$ =
  view$
    .map(parseTree)
    .switch()
    .scan((oldVnode, vnode) => patch(oldVnode, vnode), rootElem)
    
    // `rootElem` is emitted first
    .skip(1)
```